### PR TITLE
Add fp8 flash attention

### PR DESCRIPTION
**Pull request by manman-ren**
python run_benchmark.py triton --op fp8_attention --only triton_flash_v2 --metrics tflops

### CODE REVIEW

File: torchbenchmark/operators/fp8_attention/operator.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    BenchmarkOperator,
+    BenchmarkOperatorMetrics,
+    Mode as BenchmarkMode,
+    register_benchmark,
+    register_metric,
+)
+
+try:
+    # colfax Flash Attention V2 on FP8 for Hopper
+    # https://www.internalfb.com/code/fbsource/fbcode/ai_codesign/gen_ai/cutlass-kernels/src/fmha-pipeline/README.md
```
Comment (xuzhao9) [inline torchbenchmark/operators/fp8_attention/operator.py:23]: Let's remove the internal link from the PR.

**Review by xuzhao9 — APPROVED** [review]

**Comment by manman-ren** [comment]
@xuzhao9 Can you merge the change for me? I don't have permission. Thanks!

**Comment by xuzhao9** [comment]
Hi @manman-ren, we need to first import this PR as Diff. Can you please click the "import to fbsource" to import it? Then I can help stamp and land this as a Diff.

**Comment by manman-ren** [comment]
> Hi @manman-ren, we need to first import this PR as Diff. Can you please click the "import to fbsource" to import it? Then I can help stamp and land this as a Diff.

Got it. This already exists internally. Only a minor difference, the internal version has the comment mentioning internal link.

**Comment by xuzhao9** [comment]
@manman-ren Yes, with this lands, we can remove the internal version.

**Pull request closed without merging**